cmd/warm-backend-azure.go

```diff
@@ -153,6 +153,19 @@ func newWarmBackendAzure(conf madmin.TierAzure, _ string) (*warmBackendAzure, er
 		err        error
 	)
 
+	switch {
+	case conf.AccountName == "":
```

Comment:
These validations look good to me. We should do these in `TierConfigMgr.Edit` method, before we call into `newWarmBackend`.

Reply:
> These validations look good to me. We should do these in TierConfigMgr.Edit method, before we call into newWarmBackend.

Do you mean both Edit() and Add() ? it seems to me it is a good place to do the validation while initializing, lower level is always better for validation in my opinion.

Reply:
Good point. LGTM